docs/wfc3tools/calwf3.rst

```diff
@@ -301,20 +431,29 @@ _crc       UVIS calibrated, CR rejected, CTE cleaned image      :math:`e^{-}`
 ========   =================================================    ====================
 
 
-** DRZ and DRC products are produced with 'Astrodrizzle <https://www.stsci.edu/scientific-community/software/drizzlepac.html>`_ **
+**The DRZ and DRC products are produced with** 
+`AstroDrizzle <https://www.stsci.edu/scientific-community/software/drizzlepac.html>`__ 
+**which executes once** `calwf3` **completes.**
 
 
 Keyword Usage
--------------
-
-`calwf3` processing is controlled by the values of keywords in the input image headers. Certain keywords, referred to as calibration switches, are used to control which calibration steps are performed. Reference file keywords indicate which reference files to use in the various calibration steps. Users who wish to perform custom reprocessing of their data may change the values of these keywords in the `_raw` FITS file primary headers and then rerun the modified file through  `calwf3`. See the `WFC3 Data Handbook <https://www.stsci.edu/files/live/sites/www/files/home/hst/instrumentation/wfc3/_documents/wfc3_dhb.pdf>`_ for a more complete description of these keywords and their values.
-
-
-Using CRDS to update your reference files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-CRDS is the reference file management software used by STSCI for organizing and assigning reference files to datasets.
-Users can query CRDS to get the best references files for their data at hand. The following link details how you
-can use this locally : `Using CRDS to find the best reference files for your data <https://hst-crds.stsci.edu/bestrefs/>`_
+=============
+`calwf3` processing is controlled by the values of keywords in the input image headers. Certain 
+keywords, referred to as calibration switches, are used to control which calibration steps are 
+performed. Reference file keywords indicate which reference files to use in the corresponding
+calibration steps. Users who wish to perform custom reprocessing of their data may change 
+the values of these keywords in the ``_raw.fits`` file primary headers and then rerun the 
+modified file through `calwf3`. See the 
+`WFC3 Data Handbook <https://www.stsci.edu/files/live/sites/www/files/home/hst/instrumentation/wfc3/_documents/wfc3_dhb.pdf>`_ 
+for a more complete description of these keywords and their values.
+
+
+Using CRDS to Update Your Reference Files
+=========================================
+CRDS is the reference file management software used by STScI for organizing and assigning reference files to datasets.
+Users can query CRDS to get the best reference files for their data available at the time of the request. The following 
+link explains how you
+can use this facility locally: `Using CRDS to find the best reference files for your data <https://hst-crds.stsci.edu/bestrefs/>`_.
```

Comment:
A reference on how to get CRDS set up in Python/Jupyter or on the command line would be so helpful.

Reply:
A URL has been added which discusses command line use.